jerry-core/jerryscript.h

```diff
@@ -389,6 +389,11 @@ size_t jerry_parse_and_save_literals (const jerry_char_t *source_p, size_t sourc
                                       uint32_t *buffer_p, size_t buffer_size, bool is_c_format);
 
 /**
+ * External context functions.
+ */
+size_t jerry_get_context_size (void);
+size_t jerry_get_lcache_size (size_t row_count, size_t row_length);
```

Comment:
You can decide this alone for your extension. Internally, we will have 3 items, but you exports only two in the public interface. Internally you add the size of the context and lcache together, and provide it as a single item.

Why 3 items: CPU instructions has a limited immediate range for reading memory locations (especially ARM, I know intel has 32 bit offset). So if we put everything together we might frequently for the cpu to construct a read from 2 instructions (increase binary size). So I would not want the basic context grow too big.